src/service-disruptions/use-service-disruption-icon.tsx

```diff
@@ -0,0 +1,62 @@
+import {IconButtonProps} from '@atb/components/screen-header';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import React from 'react';
+import {StaticColor, TextColor} from '@atb/theme/colors';
+import {ScreenHeaderTexts, useTranslation} from '@atb/translations';
+import ServiceDisruption from '@atb/assets/svg/mono-icons/status/ServiceDisruption';
+import {useGlobalMessagesState} from '@atb/global-messages';
+import {isWithinTimeRange} from '@atb/global-messages/is-within-time-range';
+import {useNow} from '@atb/utils/use-now';
+import {ServiceDisruptionSheet} from '@atb/service-disruptions/ServiceDisruptionSheet';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+
+export const useServiceDisruptionIcon = (
+  color?: StaticColor | TextColor,
+  testID?: string,
+): IconButtonProps | undefined => {
+  const {t} = useTranslation();
+  const {findGlobalMessages} = useGlobalMessagesState();
+  const {open: openBottomSheet} = useBottomSheet();
+  const {service_disruption_url} = useRemoteConfig();
+  const hasValidServiceDisruptionUrl = !!service_disruption_url;
+  const now = useNow(2500);
+
+  if (!hasValidServiceDisruptionUrl) return undefined;
```

Comment:
Now with the `globalMessages`, maybe the button should be shown even without the service disruption url? 🤔